backend/populate_reservations.py

```diff
@@ -40,7 +38,7 @@ def guest_sort(guest_list):
             k += 1
 
 def guest_search(arr, low, high, x):
-    ''' binary search by email ''' 
```

Comment:
You can tell when someone's IDE of choice has good linting patterns and removes spaces at the end of statements <3